megamek/src/megamek/common/pathfinder/InfantryPathFinder.java

```diff
@@ -91,19 +69,19 @@ public void run(MovePath startingEdge) {
             jumpEdge.addStep(MoveStepType.START_JUMP);
             infantryPaths.addAll(generateChildren(jumpEdge));
             
+            // now that we've got all our possible destinations, make sure to try every possible rotation,
+            // since facing matters for field guns and if using the "dig in" and "vehicle cover" tacops rules.
+            for(MovePath path : infantryPaths) {
+                infantryPaths.addAll(AeroPathUtil.generateValidRotations(path));
```

Comment:
Curiosity's sake: why the `AeroPathUtil` here?

Reply:
That's where the generateValidRotations() method lives.